.github/pull_request_template.md

```diff
@@ -20,4 +25,10 @@ Please uncomment this block and take a look at this checklist if your PR is maki
 - [ ] `plotly.plot()` and `plotly.iplot()` are not used in any new/modified example
 - [ ] Hex codes for colors are not used in any new/modified example in favour of [these nice ones](https://github.com/plotly/plotly.py/issues/2192)
 
+## Code PR
+
+- [ ] I have read through the [contributing notes](https://github.com/plotly/plotly.py/blob/master/contributing.md) and understand the structure of the package. In particular, if my PR modifies code of `plotly.graph_objects`, my modifications concern the `codegen` files and not generated files.
+- [ ] I have added tests (if submitting a new feature or correcting a bug) or
+  modified existing tests.
+
```

Comment:
and docs?

Reply:
yes I will add this. It actually depends whether this checklist targets more first-time contributors, who might be discouraged if the list is too long, or whether it's for more frequent contributors.